Assets/Scripts/Controllers/MouseController.cs

```diff
@@ -165,7 +165,6 @@ void UpdateSelection()
                     mySelection.subSelection = (mySelection.subSelection + 1) % mySelection.stuffInTile.Length;
                 } while(mySelection.stuffInTile[mySelection.subSelection] == null);
             }
-            Debug.Log(mySelection.subSelection);
```

Comment:
Maybe commenting the statement out would be better so that it is ready for the next person who has to work with this code. Then later on we can clean out debug statements that are commented out i necessary.